docs/sqlml.rst

```diff
@@ -107,6 +109,66 @@ the content of "s3://bucket/to/spec.yaml" can be:
 
         YAML-based model spec is still under heavy development.
 
+
+TorchHub Integration
+--------------------
+Rikai supports creating models from the `TorchHub`_ registry. Here is the minimum SQL:
+
+    .. code-block:: sql
+
+        CREATE MODEL resnet50
+        USING "torchhub:///pytorch/vision:v0.9.1/resnet50";
+
+
+It could be expanded to the equivalent and complete SQL:
+
+    .. code-block:: sql
+
+        CREATE MODEL resnet50
+        FLAVOR pytorch
+        OPTIONS (device="cpu")
+        PREPROCESSOR 'rikai.contrib.torchhub.pytorch.vision.resnet50.pre_processing'
+        POSTPROCESSOR 'rikai.contrib.torchhub.pytorch.vision.resnet50.post_processing'
+        RETURNS array<float>
+        USING "torchhub:///pytorch/vision:v0.9.1/resnet50";
+
+
+Most models loaded via TorchHub should adopt the pytorch flavor. In practice, `FLAVOR pytorch`
+can be omitted. Usually, the `FLAVOR` keyword for specifying customized flavor.
+
+TorchHub URI is the only required part.
+    .. code-block:: verbatim
+
+        torchhub:///repo_owner/repo_name[:tag_name]/model_name
+
+
+In this case, here is the corresponding Python snippets to load the model:
+
+    .. code-block:: python
+
+        model = torch.hub.load('pytorch/vision:v0.9.1', 'resnet50', pretrained=True)
```

Comment:
Can this integration load non-pretrained model?

Reply:
I have not tested it. It is not planned in this IMPL for Torchhub registery.